scripts/install

```diff
@@ -1,63 +1,127 @@
 #!/bin/bash
 
-set -eu
+#=================================================
+# GENERIC START
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
 
-# Retrieve arguments
-app=$YNH_APP_INSTANCE_NAME
-domain=$YNH_APP_ARG_DOMAIN
-path=$YNH_APP_ARG_PATH
-admin=$YNH_APP_ARG_ADMIN
+source _common.sh
+source /usr/share/yunohost/helpers
+
+#=================================================
+# MANAGE SCRIPT FAILURE
+#=================================================
 
-# Source YunoHost helpers
-. /usr/share/yunohost/helpers
+ynh_clean_setup () {
+	read -p "key"
+	### Remove this function if there's nothing to clean before calling the remove script.
```

Comment:
As it's written here, you should remove it.

Reply:
done, forgot to remove my test steps ^^